src/NuGet.Clients/NuGet.CommandLine/NuGetCommand.resx

```diff
@@ -5526,4 +5526,30 @@ nuget-trusted-signers Remove -Name TrustedRepo</value>
   <data name="TrustedSignersCommandServiceIndexDescription" xml:space="preserve">
     <value>Service index for a repository to be trusted.</value>
   </data>
-</root>
+  <data name="SearchCommandDescription" xml:space="preserve">
+    <value>Search for packages matching search terms from a given source. If no sources are specified, all sources defined in %AppData%\NuGet\NuGet.config are used. If NuGet.config specifies no sources, uses the default NuGet feed.</value>
```

Comment:
>If NuGet.config specifies no sources, uses the default NuGet feed. [](start = 165, length = 66)

What do you think of changing `If NuGet.config specifies no sources, uses the default NuGet feed` to something like `If the NuGet.config has no sources, defaults to nuget.org`?

Reply:
The `PushCommandSourceDescription` string also refences the "default NuGet feed" as only "nuget.org" without the full service index.

---
In reply to: [275879284](https://github.com/NuGet/NuGet.Client/pull/2772#discussion_r275879284) [](ancestors = 275879284)

Reply:
I would remove this 
`If no sources are specified, all sources defined in %AppData%\NuGet\NuGet.config are used.`

It's incorrect, the configs stack. 

`If no sources are specified, the sources defined in the configs loaded from the working directory up will be used.`

Or just say the "default" sources. 

//cc @anangaur

Reply:
I prefer the "default sources". The CLI help should be terse, we can properly describe this behavior in documentation.

Reply:
`Search for packages matching search terms from a given source. If no sources are specified, the sources defined in the configurations loaded from the working directory will be used. If NuGet.config specifies no sources, uses the default NuGet feed.`

Addressed here (not ready for review): https://github.com/NuGet/NuGet.Client/pull/2834/files#diff-a15bcad9f22860661028a91cf708be67R5530